governance_rules.go

```diff
@@ -18,10 +18,12 @@ type GovernanceRulesResponse struct {
 // implement support for regex AND user/company cohort.  Logically both conditions must be true
 // also make example of regex with OR/AND conditions between expressions
 type GovernanceRule struct {
-	ID    string `json:"_id"`
-	Name  string `json:"name"`
-	Type  string `json:"type"`
-	Block bool   `json:"block"`
+	ID                string `json:"_id"`
+	Name              string `json:"name"`
+	Type              string `json:"type"`
+	Block             bool   `json:"block"`
+	ApplyTo           string `json:"applied_to"`              // "matching" or "not_matching"
+	UnidentifiedUsers bool   `json:"applied_to_unidentified"` // true if rule applies to unidentified users
```

Comment:
question: how did we load the `GovernanceRule` in go? The field is called `applied_to_unidentified`, just in case it's loading wrong field

Reply:
It's defined by the JSON struct tag